### ISSUE TITLE
new location for data files

### ISSUE THREAD

**Issue by sixty4k** [keroserene/rickrollrc#8]
was all excited to torture my new office mates with this, but sadly the files are no longer available.

any chance of an update?  (I'm happy to host if needed)

**Comment by keroserene** [comment]
Thanks for pinging! I updated the sources - should work on linux again.

I don't use mac or windows - if you or someone else feels like generating a nicely compressed rick astley gsm audio file and testing there / sending a pull request that would be lovely :)

**Comment by sixty4k** [comment]
You are awesome!  Will see what I can do about a gsm file.

**Comment by sixty4k** [comment]
`http://bootyhouse.org/lol/roll.gsm`

*however*, afplay on mac hates it.  play (installed via homebrew) plays it just fine, so I suspect a windows systems with cygwin will work, don't have one at hand to test.  

afplay plays the .s16 just fine however, PR coming shortly (after I test on a few systems)

**Comment by sixty4k** [comment]
oh yeah, gsm file created via `sox roll.s16 roll.gsm`

**Comment by GrpeApple** [comment]
original location is fixed now?

**Comment by CryptoDragonLady** [comment]
> original location is fixed now?

No, see my issue for a link to my fork which has working everything.
